pkg/kubectl/resource_printer.go

```diff
@@ -347,7 +347,30 @@ func printServiceList(list *api.ServiceList, w io.Writer) error {
 }
 
 func printMinion(minion *api.Node, w io.Writer) error {
-	_, err := fmt.Fprintf(w, "%s\t%s\n", minion.Name, formatLabels(minion.Labels))
+	var status []string
+	conditionMap := make(map[api.NodeConditionKind]*api.NodeCondition)
+	for i := range minion.Status.Conditions {
+		cond := minion.Status.Conditions[i]
+		conditionMap[cond.Kind] = &cond
+	}
+	if condition, ok := conditionMap[api.NodeReady]; ok {
+		if condition.Status == api.ConditionFull {
+			status = append(status, "Ready")
```

Comment:
I'm fine with this for now, but when we add more conditions, we should make this generic: strip off the Node prefix and conditionally prepend Not to the condition kind. Add a TODO (or fix it now, if you prefer).